### ISSUE TITLE
Example of programmatically running commands

### ISSUE THREAD

**Issue by XenonPy** [bony2023/react-terminal#333]
An example in README.md of running commands programmatically.

**Comment by carafelix** [comment]
> An example in README.md of running commands programmatically.

Did you find a way?

**Comment by XenonPy** [comment]
@carafelix Not really... there's some hacky stuff messing with the library but unreliable and slow
